Tasks/AzureFileCopyV4/AzureFileCopy.ps1

```diff
@@ -118,8 +118,20 @@ try {
         if([string]::IsNullOrEmpty($containerName) -or ($destination -ne "AzureBlob"))
         {
             $containerName = [guid]::NewGuid().ToString()
-            Create-AzureContainer -containerName $containerName -storageContext $storageContext -isPremiumStorage $isPremiumStorage
+            Create-AzureContainer -containerName $containerName -storageContext $storageContext -isPremiumStorage $isPremiumStorage  
         }
+        else
+        {
+            #checking if the containerName provided exist or not
+            $containerPresent = Get-AzureContainer -containerName $containerName -storageContext $storageContext
+
+            #creating container if the containerName provided does not exist
+            if([string]::IsNullOrEmpty($containerPresent) )
```

Comment:
since container is an object, this shouldn't be a string empty check, rather it should be a null check